Makefile

```diff
@@ -291,7 +291,7 @@ build_ship_integration_test:
 	docker build -t $(DOCKER_REPO)/ship-e2e-test:latest -f ./integration/Dockerfile .
 
 pkg/lifecycle/daemon/ui.bindatafs.go: .state/mark-ui-gitignored .state/build-deps web/app/.state/built-ui
-	go-bindata-assetfs -pkg daemon \
+	export PATH=$(GOPATH)/bin:$$PATH; go-bindata-assetfs -pkg daemon \
```

Comment:
exporting seems like an overkill.   you could just run `$GOPATH/bin/go-bindata-assetfs`

Reply:
I wish!  But go-bindata-assetfs calls go-bindata, which isn't found unless GOPATH/bin is in the path